src/tools.py

```
@@ -31,4 +32,13 @@ def handoff_to_agent(
-    return handoff_to_agent
\ No newline at end of file
+    return handoff_to_agent
+
+@tool("sequence_diagram_generator")
+def sequence_diagram_generator(architecture_output: str):
+    """
+    Converte a saída do agente de arquitetura em um diagrama de sequência PlantUML e gera uma imagem do diagrama.
+    Retorna o caminho do arquivo gerado.
+    """
+    plantuml_output = json_to_plantuml(architecture_output)
+    diagram_path = generate_diagram(plantuml_output)
```

Ruff (0.8.2)

Local variable `diagram_path` is assigned to but never used

Remove assignment to unused variable `diagram_path`

(F841)